frontend/src/components/filter/FilterOtherEntitiesSelect.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import FilterSelect from './FilterSelect';
+import { OTHER_ENTITY_TYPES } from '../../Constants';
+import { filterSelectProps } from './props';
+
+const OTHER_ENTITY_OPTIONS = OTHER_ENTITY_TYPES.map((label, value) => ({ value, label }));
+
+export default function FilterOtherEntitiesSelect({
+  onApply,
+  inputId,
+  query,
+}) {
+  const onApplyClick = (selected) => {
+    onApply(selected);
+  };
+  return (
+    <FilterSelect
+      onApply={onApplyClick}
```

Comment:
Can we just pass the `onApply` prop directly through to this component?

Reply:
I have removed it and just passed onApply.